test/gibct/containers/Modals.unit.spec.js

```diff
@@ -0,0 +1,17 @@
+import { createStore } from 'redux';
+import { expect } from 'chai';
+import React from 'react';
+import SkinDeep from 'skin-deep';
+
+import Modals from '../../../src/js/gi/containers/Modals';
+import reducer from '../../../src/js/gi/reducers';
+
+const store = createStore(reducer);
+
+describe('<Modals>', () => {
+  it('should render', () => {
+    const tree = SkinDeep.shallowRender(<Modals store={store}/>);
```

Comment:
This container is only bringing in Modal components which are already covered by common component tests. So here we are testing the Modals connected component directly (passing in store) and doing a simple render check.

In the other container tests, I am testing the undecorated component, so we need to manually pass in the props from the reducers. That said, I can refactor this test to follow the same pattern as the others for consistency.